app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListFragment.kt

```diff
@@ -529,10 +530,14 @@ class ThreadListFragment : TwoPaneFragment(), SwipeRefreshLayout.OnRefreshListen
     }
 
     private fun observeNetworkStatus() {
```

Comment:
```suggestion
    private fun observeNetworkStatus() = lifecycleScope.launch {
```

Reply:
Hum...  I'm not a big fan, cause the launch return a job and i don't think that'll be used